### PR TITLE
Project Positioning Floating Elements

### PR DESCRIPTION
**Pull request by s0kil**
#### Team:

- @lucasmazo32 
- @s0kil

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+}
+
+#inflation_demo p {
+  position: absolute;
+  font-size: 12px;
+  max-width: 180px;
+
+  /* word-break: break-all; */
+
+  /* line-break: loose; */
```
Comment (pelzolga123) [inline style.css]: Commented code needs to be deleted.

File: index.html [hunk 2]
```diff
@@ -3,10 +3,546 @@
...
+                fill-rule="evenodd"
+                clip-rule="evenodd"
+                d="M15.406 5.359L8.978 0v3.215C3.82 3.215.406 8.107.406 12.66 1.653 9.133 4.29 7.517 8.978 7.517v3.2l6.428-5.358z"
+                fill="#000"
+              />
+            </svg>
+          </a>
+        </div>
+      </div>
+    -->
```
Comment (pelzolga123) [inline index.html]: Commented code needs to be deleted.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project needs corrections.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: amazing!
But there are some changes needed: 
* [ ] Please add descriptive name and project description to your pull request. [Link to example](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
* [ ] The NY Times logo in navbar should be a link.
* [ ] All elements of navbar should be aligned properly. Right now left side is higher than others.
image
* [ ] Navbar should have a fixed position (it sticks to the top part of the screen).
* [ ] 'Theory of inflation' part is not aligned like an original one:
image
Original:
image
* [ ] Comment part should be aligned with article like in original website.
image
* [ ] This sections are missing:
image
image
* [ ] Aside parts are missing:
image
image
* [ ] Commented code needs to be deleted.

### Please kindly Fix the above issue and send another code review request.

If you have any questions or need help, you can message me on slack. *Happy coding*

**Review by mosesogwo — CHANGES_REQUESTED** [review]
## :clap: You are doing a good job.
However, you still need to address the following:

- The aside section is not properly aligned.
## **Your project:**
image

## **Original:**
image

## **Your project:**
image


## **Original:**
image

- The title of your README is for the Forms project and not this project.

- You haven't added a description to this pull request as requested in the last review.

### Kindly make these changes and submit another code review request.

### You are almost there. Keep up the good work.

**Status: Changes requested**

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi team!
### Wow! The project looks really very good! The readme file is so descriptive! :+1:  

### But still needs changes.
Kindly recommended checking HTML markup with this validator https://validator.w3.org/#validate_by_upload.

screenshot

Now it shows errors in your project


 
##### Footer

* [x] It should have a fixed position (it sticks to the bottom part of the screen) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

* Please remember that your branch name should be descriptive (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. You can read more about it [here](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). 
* Pull Request should have a meaningful title and a short description [General requirements](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/).
 I hope in the next project you can do this! image


screen

### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
Kindly recommended fix errors in HTML file, make footer sticks to the bottom part of the screen

**Comment by s0kil** [comment]
@Anna-Myzukina Hello Anna, We have tested the project once again and the footer is aligned properly.
Screenshot from 2019-12-03 18-03-55

**Comment by nikasulo** [comment]
# Great work team! :rocket: 
You may merge this PR and move on with your projects :+1: .

**Pull request merged**